e2e/_suites/ingest-manager/features/fleet_mode_agent.feature

```diff
@@ -21,6 +21,17 @@ Scenario: Stopping the agent stops backend processes
   Then the "filebeat" process is in the "stopped" state on the host
     And the "metricbeat" process is in the "stopped" state on the host
 
+@restart-agent
+Scenario Outline: Restarting a <os> agent restarts backend processes
```

Comment:
since restarting the agent and restarting the host are different can we rename the step something akin to:
Restarting a <os> host with persistent agent restarts backend processes

Reply:
I'll rephrase it, thanks!